test/specs/commonmark/commonmark.0.29.json

```diff
@@ -2766,7 +2766,7 @@
     "start_line": 6003,
     "end_line": 6007,
     "section": "Code spans",
-    "shouldFail": true
+    "shouldFail": false
```

Comment:
instead of setting `shouldFail` to `false` could you just remove the line?

Reply:
Done.